OpenRA.Mods.Common/Activities/Attack.cs

```diff
@@ -116,7 +118,8 @@ protected virtual Activity InnerTick(Actor self, AttackBase attack)
 
 			var targetedPosition = attack.Info.AttackTargetCenter ? Target.CenterPosition : Target.Positions.PositionClosestTo(pos);
 			var desiredFacing = (targetedPosition - pos).Yaw.Facing;
-			if (facing.Facing != desiredFacing)
+			if ((facingTolerance <= 1 && facing.Facing != desiredFacing) ||
+				Math.Abs((facing.Facing - desiredFacing) % 256) > facingTolerance)
```

Comment:
Still incorrect with NormalizFacing, it only wraps values between range [0, 255]. GetNearestFacing seems to have something similar to what we need